modules/RTC/BridgeChannel.js

```diff
@@ -290,11 +290,10 @@ export default class BridgeChannel {
 
             switch (colibriClass) {
             case 'DominantSpeakerEndpointChangeEvent': {
-                // Endpoint ID from the Videobridge.
-                const dominantSpeakerEndpoint = obj.dominantSpeakerEndpoint;
+                const { dominantSpeakerEndpoint, previousSpeakers = [] } = obj;
 
                 logger.info(`New dominant speaker: ${dominantSpeakerEndpoint}.`);
```

Comment:
I guess it can be useful to add the `previousSpeakers` to the log too. WDYT?

Reply:
Ok, I can make it a debug rather than info since it could be verbose. Is that ok ?

Reply:
@hristoterezov , do you think separate lists for speakers entering/leaving the previous speakers list will be helpful for the application, like we have it for endpoints entering/leaving lastN ?